OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -37,9 +37,307 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="6ra" to="46ra"/>
+                            <title type="complete" ref="LIT1828Mahale"/>
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="6ra" to="8va"/>
+                                <title type="complete" ref="LIT1828Mahale#Moses1"/>
+                                
+                                <incipit xml:lang="gez">
+                                    <locus from="6ra" to="6rb"/>
+                                    <pb n="6r"/><cb n="a"/>
+                                    <seg type="inscriptio"><hi rend="rubric">ማኅልየ፡ ነቢያት፡</hi> ወጸሎቱ፡ ለሙሴ፡</seg>
```

Comment:
I never use `<seg type="inscriptio">`
Is this a special annotation for this specific ms?

Reply:
It is an optional way of encoding a title occurring at the beginning of a work, see https://betamasaheft.eu/Guidelines/?id=ColophSupplTit (a confusing page, but a start, if you're interested). I think that we will not visualise this in any way in the catalogue, but the encoding for me did  not take much time, so I included it.